new/LIT6988PPGedewon.xml

```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6988PPGedewon" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer of ʾAbbā Gedewon</title>
+                <title xml:lang="gez" xml:id="t2">ይቤ፡ አባ፡ ጌዴዎን፡ ሶበ፡ ይወጽእ፡ ገዳመ፡ </title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Yǝbe ʾabbā Gedewon soba yǝwaṣṣǝʾ gadāma…</title>
```

Comment:
Delete '...'

Reply:
It is meant to indicate that this is the incipit, not a title proper, so I have added the same in fidal